src/styles/components/footer/footer.module.scss

```diff
@@ -32,27 +50,90 @@
     display: flex;
     flex-direction: column;
     justify-content: space-around;
+    align-items: center;
     p {
-        display: flex;
-        justify-content: center;
         text-align: center;
         line-height: 1.5;
     }
-    h3 {
-        display: flex;
-        justify-content: center;
+
+    width: 35%;
+    gap: 8px;
+    // border: red solid 2px;
+
+    @include mixins.tablet {
+        width: 100%;
+        padding-bottom: 8px;
+    }
+
+    @include mixins.phone {
+        width: 100%;
```

Comment:
can remove this since tablet already targets the width and will apply to phone styles as well